### PR TITLE
Admins can close alerts with option to notify providers

### PR DESCRIPTION
**Pull request by nobane**

### CODE REVIEW

File: _15thnight/core.py [hunk 1]
```diff
@@ -216,3 +216,26 @@ def send_help_message(user, message):
         recipients=[current_app.config.get('SUPPORT_EMAIL', '')]
     )
     queue_send_email.apply_async(kwargs=dict(message=message))
+
+
+def send_out_alert_closed(alert):
+    #TODO: Better date formatting
+    #TODO: Send out which needs were just resolved, to the outstanding providers
```
Comment (justinnoah) [inline _15thnight/core.py:223]: Perhaps these should be taken care of before merging?

Comment (nobane) [reply _15thnight/core.py:223]: Which TODO? Both?

**Comment by justinnoah** [comment]
The "Resolved at" datetime is not being shown on page (unsure if DB or JS issue).

**Comment by justinnoah** [comment]
Need to ask if we should note the admin resolved all needs. Currently all are resolved, but the advocate has no idea why.

**Comment by justinnoah** [comment]
Need to ask if we should note the admin resolved all needs. Currently all are resolved, but the advocate (or anyone looking at said alert) has no idea why.

File: CHANGELOG [hunk 2]
```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 
+## 0.1.2
```
Comment (justinnoah) [inline CHANGELOG]: This line should say "develop" or "Pre-Release" or similar. Let's NOT give version numbers until we are ready to publish said version to master and tag it.

File: _15thnight/core.py (hunk 1 above)
Comment (nobane) [reply _15thnight/core.py:223]: I am going to say we should push ahead with it as-is. I think what is currently here is fine for now, and we need to deliver something ASAP.

**Comment by nobane** [comment]
Resolved at is being set correctly now and I updated the changelog version.

File: _15thnight/core.py (hunk 1 above)
Comment (justinnoah) [reply _15thnight/core.py:223]: Due to time sensitivity, agreeing.

File: _15thnight/api/alert.py [hunk 3]
```diff
@@ -93,3 +95,23 @@ def delete_alert(id):
...
+        return api_error('Alert not found')
+
+    for need in alert.needs:
+        need.resolved = True
+        need.resolved_at = datetime.utcnow()
+        need.save(False)
+
+    alert.save()
+
+    if ('notifyProvidersAllResolved' in request.json and
```
Comment (justinnoah) [inline _15thnight/api/alert.py:113]: This does not pass as expected. Email alerts are not being sent.

**Pull request merged**